config.py

```diff
@@ -3,8 +3,19 @@
 config = ConfigParser()
 config.read("custom.conf")
 
-REQUIRED_LABELS_ANY = config.get('Labels', 'required-labels-any').split(',')
-REQUIRED_LABELS_ALL = config.get('Labels', 'required-labels-all').split(',')
-BANNED_LABELS = config.get('Labels', 'banned-labels').split(',')
+required_any = config.get('Labels', 'required-labels-any')
+required_all = config.get('Labels', 'required-labels-all')
+banned = config.get('Labels', 'banned-labels')
+
+REQUIRED_LABELS_ANY = '' if required_any == '' else required_any.split(',')
```

Comment:
these are a good candidates for the python `None` 
I would probably write it as 
```py
required_any.split(',') if required_any else None
```